canonical-web-and-design/react-components
# Only pass Slider id prop to range input, not to number input.

## Done

- Explicitly added `id` prop to `Slider`, which is then only passed to the range input.

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Edit the Slider story "With input" and add an argument for `id: "some-string"`
- Open Storybook and inspect the slider and check that only the range input has the id, not both the range and number inputs.
